package.json

```diff
@@ -16,7 +16,7 @@
     "jekyll-dev": "bundle exec jekyll serve -s jekyll --incremental --host=0.0.0.0",
     "webpack-dev": "webpack --watch --mode=development",
     "dev": "concurrently \"docker-compose up -d https\" \"yarn webpack-dev\" \"yarn jekyll-dev\"",
-    "dev-clean": "bundle exec jekyll clean",
+    "dev-clean": "bundle exec jekyll clean && rm jekyll/.jekyll-metadata",
```

Comment:
adding this to address cache issues where jekyll is regenerating all the files on each save for local dev builds.